packages/core/src/html/vueSlotSyntaxProcessor.ts

```diff
@@ -21,68 +25,67 @@ function getVslotShorthandName(node) {
 /*
  * Shifts the slot node deeper by one level by creating a new intermediary node with template tag name.
  */
-function shiftSlotNodeDeeper(node) {
-  if (!node.children) {
-    return;
-  }
+export function shiftSlotNodeDeeper(node: DomElement) {
+  const nodeChildren = node.children ?? [];
 
-  node.children.forEach((child) => {
+  nodeChildren.forEach((child) => {
     const vslotShorthandName = getVslotShorthandName(child);
     if (vslotShorthandName && child.name !== 'template') {
-      const newSlotNode = cheerio.parseHTML('<template></template>')[0];
+      const newSlotNode = cheerio.parseHTML('<template></template>')[0] as any;
```

Comment:
Is it possible for `any` to not be used here? (Or is this something that will need to wait for future typing support)

Reply:
Yes ideally we should have better typing support for this. I have updated this to be `... as unknown as T` as a little hack to force the type.